app/views/candidate_interface/gcse/grade/edit.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for :title, title_with_error_prefix(t("gcse_edit_grade.page_titles.#{@subject}"), @application_qualification.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_gcse_details_edit_type_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @application_qualification, url: candidate_interface_gcse_details_update_grade_path, method: :patch do |f| %>
+      <h1 class="govuk-heading-xl"><%= t("gcse_edit_grade.page_titles.#{@subject}") %></h1>
+
+      <%= f.govuk_error_summary %>
```

Comment:
The error summary should be above the `<h1>`. See https://design-system.service.gov.uk/components/error-summary/#where-to-put-the-error-summary.

Reply:
Resolved in 9046edb